recipes/libvpx/all/conanfile.py

```diff
@@ -102,50 +97,48 @@ def generate(self):
             # env.define("CFLAGS", "")
             # env.define("CXXFLAGS", "")
```

Comment:
- Could you move this logic with custom env right before `tc.generate(env)`?
- Why do you reset env of AutotoolsToolchain for non-msvc compilers?

Reply:
Point 1) ok.

Point 2) re -stdlib=libc++, this was here before I started editing this recipe.
I can remove it, but I can only simply watch the CI for errors, and ask you (or another Apple user) to check if it works.

Point re reset env) without it, the build fails with "Unknown option -MD, see gen_msvs_vcxproj.sh --help for available options"

Out of interest, here is what it spits out:
```
$ ./gen_msvs_vcxproj.sh
Usage: cygpath (-d|-m|-u|-w|-t TYPE) [-f FILE] [OPTION]... NAME...
       cygpath [-c HANDLE]
       cygpath [-ADHOPSW]
       cygpath [-F ID]

Convert Unix and Windows format paths, or output system path information

Output type options:

  -d, --dos             print DOS (short) form of NAMEs (C:\PROGRA~1\)
  -m, --mixed           like --windows, but with regular slashes (C:/WINNT)
  -M, --mode            report on mode of file (binmode or textmode)
  -u, --unix            (default) print Unix form of NAMEs (/cygdrive/c/winnt)
  -w, --windows         print Windows form of NAMEs (C:\WINNT)
  -t, --type TYPE       print TYPE form: 'dos', 'mixed', 'unix', or 'windows'

Path conversion options:

  -a, --absolute        output absolute path
  -l, --long-name       print Windows long form of NAMEs (with -w, -m only)
  -p, --path            NAME is a PATH list (i.e., '/bin:/usr/bin')
  -U, --proc-cygdrive   Emit /proc/cygdrive path instead of cygdrive prefix
                        when converting Windows path to UNIX path.
  -s, --short-name      print DOS (short) form of NAMEs (with -w, -m only)
  -C, --codepage CP     print DOS, Windows, or mixed pathname in Windows
                        codepage CP.  CP can be a numeric codepage identifier,
                        or one of the reserved words ANSI, OEM, or UTF8.
                        If this option is missing, cygpath defaults to the
                        character set defined by the current locale.

System information:

  -A, --allusers        use `All Users' instead of current user for -D, -O, -P
  -D, --desktop         output `Desktop' directory and exit
  -H, --homeroot        output `Profiles' directory (home root) and exit
  -O, --mydocs          output `My Documents' directory and exit
  -P, --smprograms      output Start Menu `Programs' directory and exit
  -S, --sysdir          output system directory and exit
  -W, --windir          output `Windows' directory and exit
  -F, --folder ID       output special folder with numeric ID and exit
Try `cygpath --help' for more information.
gen_msvs_vcxproj.sh: Project name (--name) must be specified!
```

Reply:
Ok, so it now will only do it when is_msvc() == True, so that should be ok?